README.md

```diff
@@ -0,0 +1,134 @@
+This is a client implementation of the Apache Camel Language Server Protocol for Emacs
+
+# Camel LSP client for Emacs
+
+Code completion for Camel LS with XML LS
+
+![Demo](images/camell.gif)
+
+NOTE: Completion is with Ctrl+Alt+i 
+
+## Prerequisites
+
+
+In a `~/.emacs.d/init.el` file:
+
+```
+(require 'package)
+(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)
+;; Comment/uncomment this line to enable MELPA Stable if desired.  See `package-archive-priorities`
+;; and `package-pinned-packages`. Most users will not need or want to do this.
+;;(add-to-list 'package-archives '("melpa-stable" . "https://stable.melpa.org/packages/") t)
+(package-initialize)
+(custom-set-variables
+ ;; custom-set-variables was added by Custom.
+ ;; If you edit it by hand, you could mess it up, so be careful.
+ ;; Your init file should contain only one such instance.
+ ;; If there is more than one, they won't work right.
+ '(package-selected-packages '(lsp-mode)))
+(custom-set-faces
+ ;; custom-set-faces was added by Custom.
+ ;; If you edit it by hand, you could mess it up, so be careful.
+ ;; Your init file should contain only one such instance.
+ ;; If there is more than one, they won't work right.
+ )
+ 
+(require 'lsp-mode)
+(add-hook 'nxml-mode-hook #'lsp)
+
+```
+In a `~/.emacs.d/lsp-camel.el` file:
+
+```
```

Comment:
You can add the language name here so that the code block benefits from syntax highlighting. see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting